Components/IModel.cs

```diff
@@ -105,6 +111,19 @@ public interface IModel
         /// </summary>
         bool HidePlaceholderMenu { get; set; }
 
+        /// <summary>
+
```

Comment:
Would not have mentioned it if it was the only thing but... weird blank line :)

Reply:
done